pkg/cache/postgres.go

```diff
@@ -58,7 +58,7 @@ func (c *postgresCache) AddCheckFromStatus(check pkg.Check, status pkg.CheckStat
 	}
 
 	if check.ID != uuid.Nil {
-		return uuid.Nil, nil
```

Comment:
@moshloop Not sure why this was set to nil uuid

Due to this being nil id, existing checks were marked deleted instead of being updated